Makefile

```diff
@@ -51,18 +51,26 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 	chmod 644 $@
 
 .PHONY: install
+install: | $(abspath $(DESTDIR)/$(PREFIX))
```

Comment:
I feel that this is needless complexity/obfuscation; just include `mkdir -p ...` as the first line under the `install` target. It will not produce any error message and will still return a `0` exit code, even if the directory already exists. It is idempotent by design.